streamlit_app.py

```diff
@@ -156,32 +189,15 @@ def main(nada_test_file_name=None, path_nada_bin=None, path_nada_json=None):
     st.code(program_code, language='python')
```

Comment:
```suggestion
    with st.expander(f"Nada Program: {program_name}"):
        st.code(program_code, language='python')
```

Reply:
Discovered you can make this collapsible. Makes it easier to go straight to the inputs instead of looking at the code. just a suggestion.

<img width="1409" alt="image" src="https://github.com/user-attachments/assets/1837aae2-71e2-46ca-8012-9010ed0f74b8">

Reply:
great point! will add this